scst-store/cluser-specific-scanner-configurations.hbs.md

```diff
@@ -0,0 +1,42 @@
+# Connecting Vulnerability scanning to SCST - Store 
+This page summarizes how to configure and connect your scanner with Supply Chain Security Tools - Store and provides links to more detailed explanations of individual topics.
+
+## Metadata Store
```

Comment:
i don't understand this section.  doesn't feel like we need it and can delete it.  let's try to keep our docs as concise as possible.